source/web/static/js/article.js

```diff
@@ -16,11 +27,14 @@ $(document).ready(function(){
 			}).addTo(mymap);
 
 	var markers = new Array();
-
+	var cleanCoords = new Array();
```

Comment:
Does "clean" mean "validated"? I'd probably name this `validatedCoords` or `validCoords` so as not to introduce a near-synonym.